components/sp-solutions/org.wso2.carbon.sp.solutions.twitter.analytics/widgets/PopularTweets/src/PopularTweets.jsx

```diff
@@ -14,62 +14,149 @@
  * limitations under the License.
  */
 
-import React, {Component} from 'react';
+import React from 'react';
 import Widget from '@wso2-dashboards/widget';
+import {MuiThemeProvider} from 'material-ui/styles/index';
+import {IconButton} from "material-ui";
+import RefreshIcon from "material-ui/svg-icons/navigation/refresh";
 import Tweet from 'react-tweet-embed'
 import './resources/tweet.css';
 import {Scrollbars} from 'react-custom-scrollbars';
 
+const MAX_DISPLAYED_TWEET_COUNT = 5;
+const MAX_UNREAD_TWEET_COUNT = 15;
+
 class PopularTweets extends Widget {
     constructor(props) {
         super(props);
 
         this.state = {
             tweetData: [],
+            unreadTweets: [],
             width: this.props.glContainer.width,
             height: this.props.glContainer.height
         };
 
         this.handleResize = this.handleResize.bind(this);
```

Comment:
Use inline function for resizing,
https://github.com/wso2/analytics-solutions/blob/master/components/sp-solutions/org.wso2.carbon.sp.solutions.http.analytics/widgets/HTTPAnalyticsLatencyComparison/src/HTTPAnalyticsLatencyComparison.jsx#L86

Reply:
Done